bids/modeling/tests/test_statsmodels.py

```diff
@@ -20,6 +23,13 @@ def graph():
     graph.load_collections(scan_length=480, subject=["01", "02"])
     return graph
 
+def test_write_graph(graph):
+    dot = graph.write_graph()
+    assert isinstance(dot, Digraph)
+    assert path.exists("graph.dot")
+    os.remove("graph.dot")
+    assert path.exists("graph.dot.png")
+    os.remove("graph.dot.png")
```

Comment:
Use a temporary directory:

```suggestion
@pytest.mark.skipif(not has_graphviz, reason="Test requires graphviz")
def test_write_graph(graph, tmp_path):
    dot = graph.write_graph(tmp_path / "graph.dot")
    assert isinstance(dot, Digraph)
    assert path.exists(tmp_path / "graph.dot")
    assert path.exists(tmp_path / "graph.dot.png")
```

Reply:
Updated.